deployer/README.md

```diff
@@ -194,6 +202,22 @@ This allows us to optimise and parallelise the automatic deployment of our hubs.
 ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
 ```
 
+### `new-grafana-token`
+This function uses the admin credentials located in `helm-charts/support/enc-support.secret.values.yaml` to check if a [Grafana Service Account](https://grafana.com/docs/grafana/latest/administration/service-accounts/) named `deployer` exists for a cluster's Grafana, and creates it if it doesn't.
+For this service account, it then generates a Grafana token named `deployer`.
+This token will be used by the [`deploy-grafana-dashboards` workflow](https://github.com/2i2c-org/infrastructure/tree/HEAD/.github/workflows/deploy-grafana-dashboards.yaml) to authenticate with Grafana’s HTTP API
```

Comment:
```suggestion
This token will be used by the [`deploy-grafana-dashboards` workflow](https://github.com/2i2c-org/infrastructure/tree/HEAD/.github/workflows/deploy-grafana-dashboards.yaml) to authenticate with Grafana’s REST API
```

I mean, probably :D

Reply:
Grafana calls it HTTP API in their docs https://grafana.com/docs/grafana/latest/developers/http_api/ and I think this is why I called it http api also?

But I went and searched if there are any differences and found out that http api it's is own thing apparently https://www.programsbuzz.com/article/what-difference-between-rest-api-and-http-api.

I will link these in the docs to have them at hand. Thanks for sparking a dive into this topic. I had no idea they are different

Reply:
Me either - that's super interesting!